README.md

```diff
@@ -91,22 +92,25 @@ We use Travis as a continuous integration service. The [Travis job](.travis.yml)
 
 ## Deployment
 
-The official and stable build lives at https://shin.gokgs.com. Each commits is also build and deployed on now.sh on:
+The official and stable build lives at https://shin.gokgs.com. Each commit is also built and deployed using [Vercel](https://www.vercel.com) (previosly called *Now*) on:
 
 ```
 https://shinkgs-BRANCH.now.sh
 ```
 
 With `BRANCH` being one of the Git branches of this repository. For example, a live version of master is always available at https://shinkgs-master.now.sh. **Warning**: the `master` branch should always work but is certainly not bullet-proof and bug-free. Use the [official build](https://shin.gokgs.com) for any sensitive games (such as tournaments).
 
-If you are developing the app and want to deploy in the could, use the [now.sh service](https://zeit.co/now):
+If you are developing the app and want to deploy in the cloud, use the [Vercel service](https://www.vercel.com):
 
 ```
-yarn global add now
-yarn now-deploy
+# Install `vercel` globally
+yarn global add vercel 
+
+# Deploy the project.
+vercel
```

Comment:
this part didn't work for me.  yarn add said it worked, but i can't find an executable called 'vercel' anywhere.

Reply:
I've added a 'Troubleshooting' section that includes some details on checking that the Yarn 'bin' directory is in the path environment variable.